### ISSUE TITLE
Render docstrings with double quotes

### ISSUE THREAD

**Issue by patrick91** [prettier/plugin-python#57]
I've been trying running prettier on the [django](https://github.com/django/django) codebase and I noticed that they use single quotes for almost all the strings and double quotes for the docstrings, which seems to be a common things in python (I haven't seen many single quotes docstrings).

**Comment by taion** [comment]
PEP 257 requires double quotes for docstrings. It also requires a blank line after the docstring for classes, but not for functions.

**Comment by rafaelcascalho** [comment]
Hey guys! I've read in the `README` that the plugin is deprecated, so I'd like to know if you're still interested in solving this issue. Case yes, I'd like to claim it :smile:

**Comment by patrick91** [comment]
@rcsm are you using this plugin? if so I'd be happy to review an merge fixes :)

**Comment by rafaelcascalho** [comment]
Yep! I've started using prettier very recently and used this plugin for a little python app of mine. Right now I'm looking for an OSS to contribute, and that's why I'd like to solve this issue :)

**Comment by patrick91** [comment]
@rcsm cool! I use black for formatting python, but could be nice to get prettier python stable anyway
